common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1985,16 +1985,16 @@ object CelebornConf extends Logging {
       .createWithDefault(256)
 
   val RPC_CACHE_CONCURRENT_LEVEL: ConfigEntry[Int] =
-    buildConf("celeborn.rpcCache.concurrent.level")
+    buildConf("celeborn.rpc.cache.concurrent.level")
```

Comment:
concurrentLevel

Reply:
Check out the spark config, I think there is no need to be concurrentLevel.

>   private[spark] val IO_COMPRESSION_ZSTD_LEVEL =
>     ConfigBuilder("spark.io.compression.zstd.level")
>       .doc("Compression level for Zstd compression codec. Increasing the compression " +
>         "level will result in better compression at the expense of more CPU and memory")
>       .version("2.3.0")
>       .intConf
>       .createWithDefault(1)

Reply:
IMO, it's not the same case. Zstd is one of the supported compression types, and `level` is a property of zstd. "concurrent level" should be a property for "cache"

Reply:
Reasonable.